modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -543,9 +555,9 @@ Status description:: error: data exception - parsing JSON failure. Underlying er
 
 // Status description:: error: data exception - unexpected map entry. Invalid input. Unexpected key `{ $key }`, expected keys are `{ $listAllowedKeys }`.
 
-// === 22NB0
+=== 22NB0
 
-// Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
+Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
```

Comment:
Not sure how you want to format this, but probably not with different quotes before and after WHERE. 

In the code it is ...evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` ... so inconsistent (I think @renetapopova wanted to go through and fix these sort of inconsistencies), but not as broken as here in the docs.

Reply:
_invalid property based access control rule involving WHERE and IS NULL_ is subcondition. 
It seems you don't use any kind of quotes or backticks in subconditions (look at them in the spreadsheet or code.)
In another PR, I already asked the question about the Cypher clauses and expressions mentioned in the messages. If I remember properly, it was decided to  postpone this discussion.

Reply:
Yes, we still have a trello card for making it more consistent. As long as the docs rendering is not broken, it should be good enough for now